android/src/main/java/com/twiliovoicereactnative/CallRecordDatabase.java

```diff
@@ -157,14 +157,18 @@ public void clear() {
 
   public CallRecord get(final CallRecord record) {
     try {
-      return callRecordList.get(callRecordList.indexOf(record));
+      synchronized (callRecordList) {
```

Comment:
Under what condition are there multiple threads accessing the call record database?

Reply:
@jhutchins Are you using new-arch? I ask because historically all invocations were coming from the same thread. IE. the main thread, and therefore impossible for a remove to happen at the exact same time as a addition.